PSIni/Functions/Out-IniFile.ps1

```diff
@@ -153,7 +153,13 @@ Function Out-IniFile {
                     }
                     else {
                         Write-Verbose "$($MyInvocation.MyCommand.Name):: Writing key: $key"
-                        Add-Content -Value "$key$delimiter$($InputObject[$key])" -Encoding $Encoding -Path $Path
+                        if ($InputObject[$key] -is [string]) {
```

Comment:
After tinkering a bit with this, I would like you to replace the code with this:

```powershell
$InputObject[$key] |
        Foreach-Object { "$key$delimiter$($InputObject[$key])" } |
        Add-Content -Value  -Encoding $Encoding -Path $Path
```

Reply:
done